am/src/nemu/isa/riscv/boot/start.S

```diff
@@ -32,12 +32,17 @@
 _start:
   init_regs
 
-  mv s0, zero
```

Comment:
Why remove this line in this file but not removed in another one?

Reply:
> Why remove this line in this file but not removed in another one?

start.S call init_regs before any jobs can start, but somehow start_flash.S does not.